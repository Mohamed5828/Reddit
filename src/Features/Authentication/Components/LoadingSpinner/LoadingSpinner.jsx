import Spinner from "react-bootstrap/Spinner";
import { SpinnerContainer } from "./LoadingSpinner.styled";

/**
 * Component to return a loading spinner 
 * @returns {React.Component}  Spinner component
 */
const LoadingSpinner = () => {
  return (
    <SpinnerContainer>
      <Spinner animation="border" />
    </SpinnerContainer>
  );
};

export default LoadingSpinner;
