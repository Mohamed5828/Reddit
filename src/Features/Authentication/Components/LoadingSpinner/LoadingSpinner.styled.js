import styled from "styled-components";


export const SpinnerContainer = styled.div`
    padding: 4px 0;
`;