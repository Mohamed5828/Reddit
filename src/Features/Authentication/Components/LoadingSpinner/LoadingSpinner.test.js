import { shallow } from "enzyme";

import LoadingSpinner from "./LoadingSpinner";

describe("LoadingSpinner Component", () => {
  it("this is a test for LoadingSpinner Component", () => {
    expect(shallow(<LoadingSpinner />)).toMatchSnapshot();
  });
});
