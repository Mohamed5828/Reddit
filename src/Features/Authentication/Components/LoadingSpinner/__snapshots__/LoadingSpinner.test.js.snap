// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner Component this is a test for LoadingSpinner Component 1`] = `
<styled.div>
  <Spinner
    animation="border"
  />
</styled.div>
`;
